SETUP/tests/ci_configuration.sh

```diff
@@ -100,23 +104,52 @@ TAG=master
 GROUP=users
 SHIFT_TO_LIVE=yes
 
-_PHP_CLI_EXECUTABLE=`which php`
```

Comment:
For this file I say we just use `command -v php` directly. Ditto `_URL_DUMP_PROGRAM` -- we control the CI env so we should just set that to `command -v wget` (or `curl` -- whichever we have in the Action) and remove the conditional bits.

Reply:
OK. Will do.